### ISSUE TITLE
Botania quest is too wordy, move it to Banners

### ISSUE THREAD

**Issue by deividaslord** [Sunekaer/academy-testing#110]
![1](https://user-images.githubusercontent.com/24681210/61594895-12a85300-abf1-11e9-8c3d-af681dde7704.PNG)

Toooo much information at once. And no one said how i made petals?
Some extra information with banners would help (maybe show how to get petals?), arrows to show where petal apothecary is.

**Comment by deividaslord** [comment]
![2](https://user-images.githubusercontent.com/24681210/61594960-8a767d80-abf1-11e9-8aad-9e0298a4756f.PNG)

maybe show this?

**Comment by Sunekaer** [comment]
Well the player is meant to search in JEI for the recipes
They will learn how to use JEI as one of the first tutorials

**Comment by aaronhowser1** [comment]
The JEI tutorial is already done, we just have to have a command reward or something to open it for the player

**Comment by aaronhowser1** [comment]
![](https://cdn.discordapp.com/attachments/552245600721174528/602305310442389520/unknown.png)
![](https://cdn.discordapp.com/attachments/552245600721174528/602305551355084801/unknown.png)
![](https://cdn.discordapp.com/attachments/552245600721174528/602305971020365844/unknown.png)
![](https://cdn.discordapp.com/attachments/552245600721174528/602306180521525279/unknown.png)

Dev art of banners for Botania room
